src/components/EventGrid/EventGrid.tsx

```diff
@@ -16,15 +16,17 @@ const EventGrid = () => {
   return (
     <Container maxWidth="xl">
       <Grid container={true} spacing={3}>
-        {events.map((event, index) => {
-          if (event.name.toUpperCase().includes(search.toUpperCase())) {
+        {events
```

Comment:
IDE was complaining about a Map function not always returning a value, so switched to filter

Reply:
A little hacky but sure lol